### ISSUE TITLE
Filebeat elasticsearch/audit fileset cannot handle request.method field

### ISSUE THREAD

**Issue by ycombinator** [elastic/beats#12297]
Observed using 6.7.2 of Elasticsearch and Filebeat.

### Steps to reproduce

1. Install Elasticsearch
2. Enable the Trial license.
2. Enable Security
3. Enable the audit log
4. Start Elasticsearch
4. Install Filebeat
5. Enable the Elasticsearch module
6. Configure the Elasticsearch module to point to the Elasticsearch audit log (JSON format)
7. Start Filebeat
8. Try to make an unauthenticated request to Elasticsearch so it fails and gets logged in the audit log

### Observed behavior

The Elasticsearch server log shows the following error:

```
[2019-05-24T22:42:54,751][DEBUG][o.e.a.b.TransportShardBulkAction] [TlwoIIJ] [filebeat-6.7.2-2019.05.24][1] failed to execute bulk item (index) index {[filebeat-6.7.2-2019.05.24][doc][TKGD7WoBKcAcowcTbnAZ], source[{"offset":83204,"log":{"file":{"path":"/Users/shaunak/elasticsearch-6.7.2/logs/elasticsearch_audit.log"}},"prospector":{"type":"log"},"source":"/Users/shaunak/elasticsearch-6.7.2/logs/elasticsearch_audit.log","message":"{\"@timestamp\":\"2019-05-24T22:42:53,148\", \"node.name\":\"TlwoIIJ\", \"node.id\":\"TlwoIIJiR6eQQXOt_54CaQ\", \"event.type\":\"rest\", \"event.action\":\"anonymous_access_denied\", \"origin.type\":\"rest\", \"origin.address\":\"[::1]:60052\", \"url.path\":\"/\", \"request.method\":\"GET\", \"request.id\":\"BR5wFGu6SF-WIDsSRbZyEw\"}","fileset":{"module":"elasticsearch","name":"audit"},"error":{"message":"field [elasticsearch.audit.user.name] doesn't exist"},"input":{"type":"log"},"@timestamp":"2019-05-24T22:42:53.148Z","elasticsearch":{"audit":{"node.id":"TlwoIIJiR6eQQXOt_54CaQ","origin_port":60052,"url.path":"/","event_type":"anonymous_access_denied","node.name":"TlwoIIJ","request.method":"GET","origin_address":"::1","layer":"rest","origin_type":"rest","request.id":"BR5wFGu6SF-WIDsSRbZyEw"}},"service":{"name":"elasticsearch"},"beat":{"hostname":"Shaunaks-MBP-2","name":"Shaunaks-MBP-2","version":"6.7.2"},"host":{"os":{"build":"18F132","name":"Mac OS X","family":"darwin","version":"10.14.5","platform":"darwin"},"name":"Shaunaks-MBP-2","id":"EF6274EA-462F-5316-A14A-850E7BFD8126","architecture":"x86_64"},"event":{"created":"2019-05-25T05:42:53.739Z","dataset":"elasticsearch.audit"}}]}
org.elasticsearch.index.mapper.MapperParsingException: Could not dynamically add mapping for field [request.method]. Existing mapping for [elasticsearch.audit.request] must be of type object but found [keyword].
```

### Expected behavior

The audit log line that caused the failed is ingested correctly into the Filebeat index by the `elasticsearch/audit` fileset.

**Comment by ycombinator** [comment]
Just trying to narrow down the versions where this error occurs. Here's what I've come up with so far:

| Prod ES | Filebeat | Monitoring ES | Error? |
| --- | --- | --- | --- |
| `8.0.0-SNAPSHOT` | `8.0.0-SNAPSHOT` | `8.0.0-SNAPSHOT` | No |
| `7.1.0` | `7.1.0` | `7.1.0` | No |
| `6.8.0` | `6.8.0` | `6.8.0` | **Yes** |
| `6.7.2` | `6.7.2` | `6.7.2` | **Yes** |

**Comment by ycombinator** [comment]
Closed via #12298.
